Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:latest
+
+# Prepare
+RUN apk add --update make perl
+COPY . /tmp
+WORKDIR /tmp
+
+# Install
+RUN perl Makefile.PL
```

Comment:
Actually, versionned `pgbadger` must be up to date. The Dockerfile should only `ADD ./pgbadger /usr/local/bin`. No `make`, no `/tmp`.